src/ovp_pipeline/commands/chat_handler.py

```
@@ -0,0 +1,794 @@
+            temperature=profile.temperature,
+            visibility=visibility,
+            topic=user_message,
+        )
+    else:
+        if chat_path is None:
+            # Find the transcript by chat_id by sweeping CHATS_DIR.
+            chat_path = _find_chat_by_id(vault, chat_id or "")
+        if chat_path is None or not chat_path.is_file():
+            raise ValueError(
+                f"chat session not found: chat_id={chat_id!r}, " f"chat_path={chat_path}"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)